srcpkgs/ntpd-rs/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'ntpd-rs'
+pkgname=ntpd-rs
+version=0.3.7
+revision=1
+build_style=cargo
+# fails on the release profile
+make_check_args="-- --skip algorithm::kalman::peer::tests::test_offset_steering_and_measurements"
+make_install_args="--path ntpd"
+depends="libcap-progs"
+short_desc="Full-featured implementation of the Network Time Protocol"
+maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
+license="Apache-2.0, MIT"
+homepage="https://github.com/pendulum-project/ntpd-rs"
+changelog="https://raw.githubusercontent.com/pendulum-project/ntpd-rs/main/CHANGELOG.md"
+distfiles="https://github.com/pendulum-project/ntpd-rs/archive/refs/tags/v${version}.tar.gz"
+checksum=e705447e61a4857875b136d5910b48f2db5dba91b77e59e430c0d651867429ad
+
+system_accounts="_ntpd_rs"
+conf_files="/etc/ntpd-rs/ntp.toml"
+alternatives="
```

Comment:
```suggestion
provides="ntp-daemon-0_1"
alternatives="
```

Reply:
You may want to add that, so it can be used as a full replacement of `openntpd|chrony|ntp`.

Reply:
Thanks, would have missed that!